pycloudflare/models.py

```diff
@@ -72,6 +72,12 @@ def create_host_zone(self, name, jump_start=False):
 
         return zone
 
+    def create_cname_zone(self, zone_name, subdomains, resolve_to):
+        host_service = self.get_host_service()
+        result = host_service.zone_set(
+            zone_name, self.user_key, subdomains, resolve_to)
+        return result
```

Comment:
As an option - we can introduce lazy access to `Zone` attributes, and allow it to be initialized with either `zone_name` or optional data